### ISSUE TITLE
Feature Request: Heat map layer of chunk change dates

### ISSUE THREAD

**Issue by bagobones** [overviewer/Minecraft-Overviewer#1665]
Looking at the slim chunk layer I was wondering if it would be possible to generate a heat map of chunk updates based on last update.

IE chunks that have been updated in X days are read and older ones would be yellow and fade out.

I think it would be a neat layer to have to highlight on order servers where people are the busiest as players tend to spread out over time and it can be hard to find new builds etc.

If this was a feature it would ideally let you set a range of color for the heat map maybe in days and look something like the slime chunk layer.

**Comment by enaut** [comment]
Sounds great! Do a fork and then a pull request ;-) I guess the maintainers will include it or help you get it to a state where it can be included.

**Comment by Wunkolo** [comment]
This would be really cool actually. Some parts of my map are more "dead" than others and whenever I push a map update, people would have to swim around the map to see what changed. This would allow for people to get right to seeing all the new tiles that updated since the last render

**Comment by bagobones** [comment]
@Tirpitz93  A history filter would require A LOT of tile data to be saved. but is also a cool idea.

What I am proposing is something that could run based off the existing delta scan, over-viewer should already be tracking when each chunk was updated somewhere. If I have time to look I assume the last update stamp shouldn't be much harder than finding the slime chunk info (unless that is just calculated on the seed).

I don't have a lot of time to look at code and python is more of a minor hobby for me.

**Comment by Tirpitz93** [comment]
@bagobones I dont think that any change to the MC side of things would need to be carried out, just the web frontend, using something like rsnapshot or other rsync based tool. only new tiles are added each time.
I already use rsnapshot to offload the map to the render server before I run overviewer.
the only requirement would be to keep the old timestamped tiles to allow rollback.

**Comment by Tirpitz93** [comment]
@bagobones I amgoing to open a feature request and fork see what I can come up with.

**Comment by generrosity** [comment]
Each chunk had an idea of "inhabited time", which I think might be a viable indication of how busy a chunk had been. Might not be worth what was asked, but might be trivial to impliment
